src/FileServiceManager/FileService/Helpers/RestException.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Net;
+using System.Runtime.Serialization;
+
+namespace FileService.Helpers
+{
+    public class RESTException : Exception
```

Comment:
Didn't saw the benefit of this class, let us discuss one more time

Reply:
The benefit is that we can throw this type of exception and send StatusCode for example with it, which can be catched with ErrorHandlingMiddlware and then send to user. If we delete this exception, what do we throw? Do all exceptions need to be sent with InternalServerError in ErrorHandlingMiddlware?

Reply:
no, you simply map all exceptions that you use to appropriate status code, see the example below

![image](https://user-images.githubusercontent.com/65189378/109014263-68968400-76cd-11eb-9262-1b153aec6607.png)

Reply:
++